randovania/games/prime1/gui/preset_settings/prime_teleporters_tab.py

```diff
@@ -0,0 +1,182 @@
+from __future__ import annotations
+
+import copy
+import dataclasses
+from typing import TYPE_CHECKING
+
+from randovania.game_description.db.dock_node import DockNode
+from randovania.gui.generated.preset_teleporters_prime1_ui import Ui_PresetTeleportersPrime1
+from randovania.gui.lib import signal_handling
+from randovania.gui.lib.node_list_helper import NodeListHelper
+from randovania.gui.preset_settings.preset_teleporter_tab import PresetTeleporterTab
+from randovania.layout.lib.teleporters import (
+    TeleporterList,
+    TeleporterShuffleMode,
+)
+
+if TYPE_CHECKING:
+    from PySide6 import QtWidgets
+
+    from randovania.game_description.db.area import Area
+    from randovania.game_description.db.node_identifier import NodeIdentifier
+    from randovania.game_description.game_description import GameDescription
+    from randovania.games.common.prime_family.layout.lib.prime_trilogy_teleporters import (
+        PrimeTrilogyTeleporterConfiguration,
+    )
+    from randovania.games.prime1.layout.prime_configuration import PrimeConfiguration
+    from randovania.gui.lib.window_manager import WindowManager
+    from randovania.interface_common.preset_editor import PresetEditor
+    from randovania.layout.preset import Preset
+
+
+class PresetTeleportersPrime1(PresetTeleporterTab, Ui_PresetTeleportersPrime1, NodeListHelper):
```

Comment:
as far as I'm concerned, `Teleporter` *is* a game-specific term for the prime series

Reply:
Hmmm renaming that to elevator again will come with a pretty bad disadvantage: It is using the common part of the ui. It will be very mixed between elevator naming and the common things like `teleporter_mode_to_description` or `teleporters_source_layout` etc. which in my opinion makes it even worse.